canonical/ubuntu.com
# fix: add conditions for minimum timedelta in scheduling

## Done

- Should not be possible to schedule exams within 30 mins 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Purchase an exam
- Try to schedule it within 30 mins
- Should get an error 

## Issue / Card

Fixes [#WD-7947](https://warthogs.atlassian.net/browse/WD-7947)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
